src/global/interfaces/link-button-properties.ts

```diff
@@ -113,7 +113,9 @@ function getLinkAttributeList(linkProperties: LinkProperties): Record<string, st
 function hostProperties(role: string, disabled: boolean): Record<string, string> {
   return Object.assign(
     { role, dir: getDocumentWritingMode() },
-    disabled ? { 'aria-disabled': 'true' } : { tabIndex: '0' },
+    disabled
+      ? { 'aria-disabled': 'true', tabIndex: '-1' }
+      : { 'aria-disabled': 'false', tabIndex: '0' },
```

Comment:
Could you check if this output generates the same as before?
```suggestion
    disabled
      ? { 'aria-disabled': 'true', tabIndex: false }
      : { 'aria-disabled': false, tabIndex: '0' },
```